en/advanced_config/compass_power_compensation.md

```diff
@@ -35,7 +35,7 @@ Performing this power compensation is advisable only if all the following statem
    - Slowly lower the throttle down to zero
    - Disarm the vehicle
    > **Note** Perform the test carefully and closely monitor the vibrations.
-1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/master/src/lib/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
+1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/2c3441aa905c0a0cc34aac3850f809be3e12a5e0/src/modules/sensors/vehicle_magnetometer/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
```

Comment:
```suggestion
1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/master/src/modules/sensors/vehicle_magnetometer/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
```

@elifBalci the link I sent you on Slack works but in fact for the docs we should be pointing to the latest version of the script.

Reply:
Right, thanks for the fix. @RicardoM17